sklearn/hpsearch/README.md

```diff
@@ -0,0 +1,34 @@
+# Parallelized Hyperparameter Search with Google Container Engine
+
+## Introduction
+
+This sample package helps you run `scikit-learn`'s `GridSearchCV` and `RandomizedSearchCV` on [Google Container Engine](https://cloud.google.com/container-engine/).
+
+
+## Requirements
+
+You will need a Google Cloud Platform project which has the following products enabled:
+
+- [Google Container Registry](https://cloud.google.com/container-registry/)
+
+- [Google Container Engine](https://cloud.google.com/container-engine/)
+
+- [Google Cloud Storage](https://cloud.google.com/storage/)
+
+
+In addition, to follow the steps of the sample we recommend you work in a [Jupyter notebook](https://jupyter.org/) running [Python](https://www.python.org/) v2.7.10 or newer.
+
+
+## Before you start
+
+1. Install [Google Cloud Platform SDK](https://cloud.google.com/sdk/downloads).
+
+1. Install [kubectl](https://cloud.google.com/container-engine/docs/quickstart).
+
+1. `git clone https://github.com/GoogleCloudPlatform/ml-on-gcp.git`
```

Comment:
Tell user to "Run the following commands"

Reply:
done